src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -177,6 +201,7 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             }
         }
 
+        [ActiveIssue(7812, TestPlatforms.Windows)]
```

Comment:
The last comment in this bug is saying this is an OS problem. Can we just run the test for OS versions that have the fix?